diff -urN xtrader.orig/Makefile.linux xtrader/Makefile.linux
--- xtrader.orig/Makefile.linux	2004-11-30 12:53:32.933635488 +0100
+++ xtrader/Makefile.linux	2004-11-30 12:57:20.118098200 +0100
@@ -1,12 +1,12 @@
 COMP=			g++
-IFLAGS=			-I../util -Isrc -I/usr/local/include -I/usr/include/ptypes
-CFLAGS=			-march=i586 -O3 -Wall -Wno-format-y2k
-LFLAGS=			-L/usr/local/lib -lz -lfltk -lptypes  -lexpat -L/usr/X11R6/lib -lm -lXpm -lX11 -ldl -lpthread -lXft
-LFLAGS2=		-L/usr/local/lib -lfltk_ -lptypes -L/usr/X11R6/lib -lXpm -lX11 -ldl -lpthread
+IFLAGS=			-I../util -Isrc -I/usr/include/ptypes
+CFLAGS=			$(OPT) 
+LFLAGS=			-L/usr/lib -lz -lfltk -lptypes  -lexpat -L/usr/X11R6/lib -lm -lXpm -lX11 -ldl -lpthread -lXft
+LFLAGS2=		-L/usr/lib -lfltk_ -lptypes -L/usr/X11R6/lib -lXpm -lX11 -ldl -lpthread
 XTRADER=		xtrader
 XTRADER2=		xtrader.static
 
-all:	$(XTRADER) $(XTRADER2)
+all:	$(XTRADER)
 
 XTRADER_OBJ=../util/MH.o ../util/MHCSVFile.o \
 		../util/MHDate.o ../util/MHDayChart.o ../util/MHDC.o \
